code/modules/jobs/job_types/roguetown/garrison/gatemaster.dm

```diff
@@ -53,7 +53,7 @@
 	beltr = /obj/item/ammo_holder/quiver/arrows
 	backr = /obj/item/storage/backpack/rogue/satchel/black
 	backl = /obj/item/gun/ballistic/revolver/grenadelauncher/bow
-	backpack_contents = list(/obj/item/storage/keyring/gatemaster = 1, /obj/item/rogueweapon/huntingknife/idagger/steel/special = 1, /obj/item/rope/chain = 1)
```

Comment:
I don't particularly think they should be getting the MAA's keyring, if we want the gatemaster to have better keys.